packages/fhir-converter/src/templates/cda/Resources/Composition.hbs

```diff
@@ -84,6 +84,12 @@
                             "status":"generated",
                             "div":"<div xmlns=\"http://www.w3.org/1999/xhtml\">{{this.section.code.displayName}}</div>",
                         }
+                    {{else}}
+                        "text":
+                        {
+                            "status":"generated",
+                            "div":"<div xmlns=\"http://www.w3.org/1999/xhtml\">unknown</div>",
```

Comment:
div cannot be empty inside

Reply:
text is optional - why did the original code not work?